### PR TITLE
chore: Scripts were added to simplify the work environment creation

### PR DESCRIPTION
**Pull request by t-karasova**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/interactive-tutorials/user_environment_setup.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# get the project_id from gcolud config
```
Comment (kweinmeister) [inline samples/interactive-tutorials/user_environment_setup.sh:3]: gcloud config

File: samples/interactive-tutorials/user_environment_setup.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```
Comment (kweinmeister) [inline samples/interactive-tutorials/user_environment_setup.sh:1]: Please add copyright headers to both files

File: samples/interactive-tutorials/user_environment_setup.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  done
+
+# uppload your service account key file
+service_acc_email=$service_account_id"@"$project_id".iam.gserviceaccount.com"
+gcloud iam service-accounts keys create ~/key.json --iam-account $service_acc_email
+
+# activate the service account using the key
+gcloud auth activate-service-account --key-file ~/key.json
+
+# set the key as GOOGLE_APPLICATION_CREDANTIALS
```
Comment (kweinmeister) [inline samples/interactive-tutorials/user_environment_setup.sh:26]: Change to CREDENTIALS (on 2 lines)

File: samples/interactive-tutorials/user_environment_setup.sh [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+# create service account (your project_id+timestamp)
+gcloud iam service-accounts create $service_account_id
+
+# assign needed roles to your new service account
+for role in {retail.admin,storage.admin,bigquery.admin}
+  do
+    gcloud projects add-iam-policy-binding $project_id --member="serviceAccount:"$service_account_id"@"$project_id".iam.gserviceaccount.com" --role="roles/${role}"
+  done
+
+# uppload your service account key file
```
Comment (kweinmeister) [inline samples/interactive-tutorials/user_environment_setup.sh:19]: upload

File: samples/interactive-tutorials/user_import_data_to_catalog.sh [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd product
+output=$(python setup_product/products_create_gcs_bucket.py)
+
+temp="${output#*The gcs bucket }"
+bucket_name="${temp% was created}"
```
Comment (kweinmeister) [inline samples/interactive-tutorials/user_import_data_to_catalog.sh:7]: Is there supposed to be just one %?

**Review by kweinmeister — CHANGES_REQUESTED** [review]

File: samples/interactive-tutorials/user_environment_setup.sh (hunk 1 above)
Comment (t-karasova) [reply samples/interactive-tutorials/user_environment_setup.sh:3]: fixed

File: samples/interactive-tutorials/user_import_data_to_catalog.sh (hunk 5 above)
Comment (t-karasova) [reply samples/interactive-tutorials/user_import_data_to_catalog.sh:7]: exactly

**Review by kweinmeister — APPROVED** [review]

**Review by chenlei1216 — APPROVED** [review]

**Review by parthea — APPROVED** [review]

**Pull request merged**